.github/workflows/auth-tests.yml

```diff
@@ -53,6 +53,7 @@ jobs:
           git update-index -q --refresh
           ## If there are changes: comment, commit, PR
           if ! git diff-index --quiet HEAD --; then
+            git branch --set-upstream-to=$GITHUB_USER/zaproxy-website
```

Comment:
Shouldn't the upstream be remote/branch?
```
git branch --set-upstream-to=origin/$BRANCH
```

An alternative is to always commit and push, and then create a PR only if it doesn't exist. Something like:

```
git add .
git commit -sm 'Authentication Tests Update'
git push -u origin $BRANCH -f
gh pr view || gh pr create -f
```

One more option (that I'm not sure of, but should work in theory) is to simply set the `fetch-depth` in the checkout step to 0. That way, we should fetch all branches upon clone and checking out the $BRANCH should automatically set the upstream.

Reply:
Yes I think you're right it should be remote/branch. I'll tweak that shortly.

Reply:
Fixed